### PR TITLE
Create QE dedicated AWS cluster profiles

### PR DESCRIPTION
**Pull request by yunjiang29**
After discuss with DPP and Installer dev team, QE should use our dedicated cluster profiles.

release https://github.com/openshift/release/pull/26984

| Profile      | Profile String       | Cloud Type |
| ---      | ---       | ---      |
| ClusterProfileAWSQE | aws-qe         | aws         |
| ClusterProfileAWSC2SQE     | aws-c2s-qe        | aws-c2s        |
| ClusterProfileAWSSC2SQE     | aws-sc2s-qe        | aws-sc2s        |
| ClusterProfileAWSChinaQE     | aws-china-qe        | aws-china        |
| ClusterProfileAWSGovCloudQE     | aws-usgov-qe        | aws-usgov        |

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianlinliu PTAL, thanks.

cc @aleskandro for arm test, as discussed with DPP,  we should use our own profiles to run QE's test.

**Comment by yunjiang29** [comment]
@petr-muller can you review and approve it? thanks

File: pkg/api/types.go
Comment (petr-muller) [inline pkg/api/types.go:1404]: Missing `aws-qe`?

Comment (yunjiang29) [reply pkg/api/types.go:1404]: @petr-muller `aws-qe` is the cluster profile of QE, the cluster type is the same `aws` profile which is  **aws**

File: pkg/prowgen/podspec.go
Comment (yunjiang29) [reply pkg/prowgen/podspec.go:296]: As we discussed, added all QE's profiles here.

File: pkg/api/types.go
Comment (petr-muller) [reply pkg/api/types.go:1404]: Does `aws-qe` have separate leases?

Comment (yunjiang29) [reply pkg/api/types.go:1404]: yes, it does, https://github.com/openshift/release/pull/27033/files#diff-aa86f662b34188da9ad50af7c05ef19dfa48af3c726cd882949002cf9112eed9R28-R30

Comment (yunjiang29) [reply pkg/api/types.go:1404]: I thought lease string is handled by [1] via cluster profile name, is `aws-qe` required here? but it's not a _cluster type_ , I'm a little confused

[1] https://github.com/openshift/ci-tools/pull/2724/files#diff-dfee74d02fd350fb1ee1ffbbbc32acf4cb075c9ae42b6e31c8c7acefe7fed623R1306-R1315

Comment (yunjiang29) [reply pkg/api/types.go:1404]: @petr-muller is there anything that I need to update before getting approval?

Comment (petr-muller) [reply pkg/api/types.go:1404]: STOP PINGING ME

Comment (petr-muller) [reply pkg/api/types.go:1404]: I review code in batches at my convenience. Stop triggering a million notifications for me. I am aware of the PR and will respond. I will not respond on-demand.

**Review by petr-muller — APPROVED** [review]

File: pkg/api/types.go
Comment (yunjiang29) [reply pkg/api/types.go:1404]: I apologize for disturbing you with my message, thanks for reviewing my PRs.

**Pull request merged**
